R/rinstall.r

```diff
@@ -15,22 +15,17 @@
 rinstall = function(pkgs = character(0L), add = FALSE, ...) {
   pkgs = lapply(pkgs, stringToPackage)
   assertFlag(add)
-
-  # FIXME: Do we need to specify the lib_path?
-
   pn = extract(pkgs, "name")
   is.cran = vlapply(pkgs, inherits, "PackageCran")
   if (any(is.cran)) {
     messagef("Installing %i packages from CRAN: %s", sum(is.cran), collapse(pn[is.cran]))
-    remotes::install_cran(pn[is.cran], build_opts = getDefaultBuildOpts(remotes::install_cran, "cran"), ...)
```

Comment:
Is there a reason why this was deleted?

Reply:
It was some time ago, but I think this was for simplicity. If you think the former line is better, please restore.

Reply:
I changed the way one can influence the build opts so it looks more clean hopefully